apps/web/src/app/reconnect/page.module.css

```
@@ -0,0 +1,163 @@
+@keyframes fadeIn {
```

Stylelint (17.3.0)

[error] Expected keyframe name "fadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)